nss-day-cohort-22/Class-Website
# Kn design

## Description of Proposed Changes
added formatting to the site, inspired by the colors of the C# logo. 

Added a binary image as the background, but am not attached to it, if someone has a different idea. Was just trying to make the site look a little different from 21 (There's a second binary image in the image folder if anyone thinks that one looks better.)

Buttons will only appear if that student has info in the JSON file. For instance,  each of the contact icons will only appear if there is a url for that site (or email address). Since the contact icons appear both on the main page and on the modal, the "Learn More" button will only appear if the student has a bio.

Also, I removed the section that describes what NSS is. It felt out of place to me since the people who are going to be visiting the class website are going to be directed there through NSS webpages so they already know what NSS is. If others think it's a good idea, we can add it back in.


## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout kn-design

```
checkout the formatting and let me know if there are any changes that we should make. There may be some spacing things from moving stuff around that I may have missed.

## Mentions 

@Chewieez @john-dulaney @Gward2489
